src/routeDefinitions.js

```diff
@@ -9,7 +9,7 @@ export default {
   FAQ_ARTICLE: '/static-faq/article',
   KIWI_LOGIN: '/kiwi-login',
   FORGOTTEN_PASSWORD: '/forgotten-password',
-  UPCOMING_BOOKING: '/upcoming-booking',
+  NEAREST_BOOKING: '/upcoming-booking',
```

Comment:
I think you should rename the actual path to `/nearest-booking` as well.

Reply:
fixed